### ISSUE TITLE
group movement

### ISSUE THREAD

**Issue by ilteroi** [LoneGazebo/Community-Patch-DLL#9033]
@nobubblegums you know there is PositionUnitsAroundTarget()? that's what the AI uses to move armies. did you take a look at that? may not be perfect either but if there are issues they should be fixed so that humans and AI both benefit.

**Comment by nobubblegums** [comment]
I went through some other AI code when I was starting out, but didn't know about that! Those functions get a bit hard to follow for me though, so I opted for the easier route. I won't be able to do much modding anymore, but I can test those if I have the time.
